fhir-path/src/main/java/com/ibm/fhir/path/function/ContainsFunction.java

```diff
@@ -36,9 +35,7 @@ public int getMaxArity() {
     
     @Override
     public Collection<FHIRPathNode> apply(EvaluationContext evaluationContext, Collection<FHIRPathNode> context, List<Collection<FHIRPathNode>> arguments) {
-        if (!hasStringValue(context)) {
-            return empty();
-        }
+        checkStringValue(context);
```

Comment:
Most (all?) of the function definitions say that if the input collection is empty, then the result must be empty as well (not an error).  Please make sure we're handling this case.

Reply:
looks good in the new common parent; I left a separate comment for the toString function